MPSGameLift/Documentation/GameLift.md

```diff
@@ -0,0 +1,140 @@
+# MultiplayerSample Project with Amazon GameLift
+
+This README covers optional setup, testing and running on [Amazon GameLift](https://aws.amazon.com/gamelift/), an AWS service to make hosting and scaling game servers easier. 
+
+## Running with [Amazon GameLift](https://docs.aws.amazon.com/gamelift/index.html)
+
+### Setup
+1. Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), if you don't already have it. You will need it to interact with Amazon GameLift.
+
+    a. Confirm installation by running `aws --version`
+    ```sh
+    C:\> aws --version
+    aws-cli/2.10.0 Python/3.11.2 Windows/10 exe/AMD64 prompt/off
+    ```
+1. Enable the "AWSGameLift" and "MPSGameLift" gems
+
+    Option 1: Commandline
+    ```sh
+    <path-to-o3de-engine>\scripts\o3de.bat enable-gem -pp <path-to-multiplayer-sample> -gn MPSGameLift
+    ```
+    Option 2: Project Manager
+    1. Open Project Manager
+    2. Select the "Configure Gems" options for Multiplayer Sample
+    3. Enable "AWSGameLift" and "MPSGameLift" gems
+
+        ![Enable GameLift Gems](Media/enable_gamelift_gems.jpg)
+    4. Click "Save"
+
+1. Build the server, game launchers, and asset bundler for MultiplayerSample
+    ```sh
+    cmake --build build\windows --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher AssetBundler --config profile -- /m /nologo
+    ```
+1. Build all the assets
+    ```sh
+    cmake --build build\windows --target MultiplayerSample.Assets --config profile -- /m /nologo
+    ```
+1. Work in progress (WiP) step: Add your AWS region to Config/default_aws_resource_mappings.json (example: "Region": "us-west-2")
+
+    a. Currently needed otherwise when the client initializes GameLift there will be an error about not having a region.
+
+    b. This step will be removed once we properly parse the game-session data which contains the fleet-id, region-id, etc  
+
+## Build Server for Windows
+1. Build Monolithic Server
+
+    a. cmake -B build\windows_mono -S . -G "Visual Studio 16" -DLY_MONOLITHIC_GAME=1 -DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0
+
+    b. cmake --build build\windows_mono --target MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher --config profile -- /m /nologo
```

Comment:
The `/nologo` argument can be removed here and the command can be placed in backticks as well.
```suggestion
    b. `cmake --build build\windows_mono --target MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher --config profile -- /m`
```

Reply:
Good call